VRDR/DeathRecord.cs

```diff
@@ -1466,1339 +1448,663 @@ public string ContributingConditions
                 }
             }
         }
 
-        /// <summary>Cause of Death Part I, Line h.</summary>
-        /// <value>the seventh underlying cause of death literal.</value>
+        /// <summary>Conditions that resulted in the cause of death. Corresponds to part 1 of item 32 of the U.S.
+        /// Standard Certificate of Death.</summary>
+        /// <value>Conditions that resulted in the underlying cause of death. An array of tuples (in the order they would
+        /// appear on a death certificate, from top to bottom), each containing the cause of death literal (Tuple "Item1") the
+        /// approximate onset to death (Tuple "Item2"), and an (optional) Dictionary representing a code, containing the following key/value pairs:
+        /// <para>"code" - the code describing this finding</para>
+        /// <para>"system" - the system the given code belongs to</para>
+        /// <para>"display" - the human readable display text that corresponds to the given code</para>
+        /// </value>
         /// <example>
         /// <para>// Setter:</para>
-        /// <para>ExampleDeathRecord.COD1H = "example";</para>
+        /// <para>Tuple&lt;string, string, Dictionary&lt;string, string&gt;&gt;[] causes =</para>
+        /// <para>{</para>
+        /// <para>    Tuple.Create("Example Immediate COD", "minutes", new Dictionary&lt;string, string&gt;(){ {"code", "1234"}, {"system", "example"} }),</para>
+        /// <para>    Tuple.Create("Example Underlying COD 1", "2 hours", new Dictionary&lt;string, string&gt;()),</para>
+        /// <para>    Tuple.Create("Example Underlying COD 2", "6 months", new Dictionary&lt;string, string&gt;()),</para>
+        /// <para>    Tuple.Create("Example Underlying COD 3", "15 years", new Dictionary&lt;string, string&gt;())</para>
+        /// <para>};</para>
+        /// <para>ExampleDeathRecord.CausesOfDeath = causes;</para>
         /// <para>// Getter:</para>
-        /// <para>Console.WriteLine($"Cause: {ExampleDeathRecord.COD1H}");</para>
+        /// <para>Tuple&lt;string, string&gt;[] causes = ExampleDeathRecord.CausesOfDeath;</para>
+        /// <para>foreach (var cause in causes)</para>
+        /// <para>{</para>
+        /// <para>    Console.WriteLine($"Cause: {cause.Item1}, Onset: {cause.Item2}, Code: {cause.Item3}");</para>
+        /// <para>}</para>
         /// </example>
-        [Property("COD1H", Property.Types.String, "Death Certification", "Cause of Death Part I, Line h.", false, "http://build.fhir.org/ig/HL7/vrdr/StructureDefinition-VRDR-Cause-Of-Death-Condition.html", false, 100)]
-        public string COD1H
+        [Property("Causes Of Death", Property.Types.TupleCOD, "Death Certification", "Conditions that resulted in the cause of death.", true, IGURL.CauseOfDeathPathway, true, 50)]
+        [FHIRPath("Bundle.entry.resource.where($this is Condition).where(onset.empty().not())", "")]
+        public Tuple<string, string /*, Dictionary<string, string>*/>[] CausesOfDeath
```

Comment:
Remove comment Dictionary<string, string>

Reply:
I left the Tuple in for now....may be useful.  Pending Pete Review.